### PR TITLE
moonpay: enable erc20 tokens

### PR DESCRIPTION
**Pull request by benma**

### CODE REVIEW

File: backend/exchanges/moonpay.go [hunk 1]
```diff
@@ -36,7 +36,16 @@ var moonpayAPICryptoCode = map[coin.Code]string{ // -> moonpay crypto currency c
 	coin.CodeBTC: "btc",
 	coin.CodeLTC: "ltc",
 	coin.CodeETH: "eth",
-	// TODO: ERC20 tokens; figure out eth-erc20- prefix vs frontend
+
+	"eth-erc20-usdt":      "usdt",
+	"eth-erc20-usdc":      "usdc",
+	"eth-erc20-bat":       "bat",
+	"eth-erc20-dai0x6b17": "dai",
```
Comment (x1ddos) [inline backend/exchanges/moonpay.go:43]: What about `eth-erc20-sai0x89d2` or is it unsupported?

File: backend/exchanges/moonpay.go [hunk 2]
```diff
@@ -36,7 +36,16 @@ var moonpayAPICryptoCode = map[coin.Code]string{ // -> moonpay crypto currency c
...
+	"eth-erc20-link":      "link",
+	"eth-erc20-mkr":       "mkr",
+	"eth-erc20-zrx":       "zrx",
+	"eth-erc20-wbtc":      "wbtc",
+	"eth-erc20-paxg":      "paxg",
```
Comment (x1ddos) [inline backend/exchanges/moonpay.go:48]: I guess this list needs to be in sync with `erc20Tokens` in backend/erc20.go? There are many instances of this list in the backend code...

**Review by x1ddos — APPROVED** [review]
LGTM

File: backend/exchanges/moonpay.go (hunk 1 above)
Comment (benma) [reply backend/exchanges/moonpay.go:43]: yes, unsupported. afaik it's the predecessor of DAI.

File: backend/exchanges/moonpay.go (hunk 2 above)
Comment (benma) [reply backend/exchanges/moonpay.go:48]: correct. could put them in consts like BTC, TBTC, etc. as well I guess.

**Pull request merged**